sunpy/net/helio/parser.py

```diff
@@ -37,17 +39,21 @@ def webservice_parser(service='HEC'):
     --------
     >>> from sunpy.net.helio import parser
     >>> parser.webservice_parser()  # doctest: +REMOTE_DATA
-    ['http://msslkz.mssl.ucl.ac.uk/helio-hec/HelioService',
-     'http://festung3.oats.inaf.it:8080/helio-hec/HelioService',
-     'http://festung1.oats.inaf.it:8080/helio-hec/HelioService',
-     'http://hec.helio-vo.eu/helio_hec/HelioService',
-     'http://msslkz.mssl.ucl.ac.uk/helio-hec/HelioLongQueryService',
-     'http://festung3.oats.inaf.it:8080/helio-hec/HelioLongQueryService',
-     'http://festung1.oats.inaf.it:8080/helio-hec/HelioLongQueryService',
-     'http://hec.helio-vo.eu/helio_hec/HelioLongQueryService']
+    ['http://helio.mssl.ucl.ac.uk/helio-hec/HelioService',
+    'http://msslkk.mssl.ucl.ac.uk/helio-hec/HelioService',
+    'http://voparis-helio.obspm.fr/helio-hec/HelioService',
+    'http://hec.helio-vo.eu/helio-hec/HelioService',
+    'http://helio.mssl.ucl.ac.uk/helio-hec/HelioLongQueryService',
+    'http://msslkk.mssl.ucl.ac.uk/helio-hec/HelioLongQueryService',
+    'http://voparis-helio.obspm.fr/helio-hec/HelioLongQueryService',
+    'http://hec.helio-vo.eu/helio-hec/HelioLongQueryService']
     """
-    link = RL.LINK + '/' + service.lower()
-    xml = link_test(link)
+    xml = None
+    for REG_LINK in REG_LINKS:
+        link = REG_LINK + service.lower()
+        xml = link_test(link)
+        if xml:
+            break
     if xml is None:
         return None
```

Comment:
I feel like this logic could be made simpler and nicer.

Reply:
it'll suffice

Reply:
I rather do better than, "it'll suffice".